### PR TITLE
Loc 722 visual issues ie

### PR DESCRIPTION
**Pull request by ChrisPam**

### CODE REVIEW

File: src/components/homes/common/RoomSpaceInformationBox/RoomSpaceInformationBox.jsx [hunk 1]
```diff
@@ -3,54 +3,54 @@ import PropTypes from 'prop-types';
...
+      }
+
+      {size &&
+        <div>
+          <img src="/images/icon-review/icon-size.png" alt="icon-size" />
+          <p>{size} m2</p>
+        </div>
+      }
+
+      {bathroom && bathroom.toString() &&
```
Comment (iludart) [inline src/components/homes/common/RoomSpaceInformationBox/RoomSpaceInformationBox.jsx]: This throws if bathroom is missing.

File: src/components/homes/common/RoomSpaceInformationBox/RoomSpaceInformationBox.jsx [hunk 2]
```diff
@@ -3,54 +3,54 @@ import PropTypes from 'prop-types';
...
+        <div>
+          <img src="/images/icon-review/icon-bathroom.png" alt="icon-bathroom" />
+          <p>{bathroom} {bathroom === 1 ? 'Bathroom' : 'Bathrooms'}</p>
+        </div>
+      }
+
+      {bathroom && bedrooms.toString() &&
```
Comment (iludart) [inline src/components/homes/common/RoomSpaceInformationBox/RoomSpaceInformationBox.jsx]: Same as above.

File: src/components/hotels/search/ResultsHolder.jsx [hunk 3]
```diff
@@ -33,12 +33,12 @@ function ResultsHolder(props) {
 
   return (
     <div className="results-holder">
-      <ReactCSSTransitionGroup
+      {/* <ReactCSSTransitionGroup
```
Comment (iludart) [inline src/components/hotels/search/ResultsHolder.jsx]: Why is this commented? Do we have a task to remove this feature?

Comment (ChrisPam) [reply src/components/hotels/search/ResultsHolder.jsx]: Because this isn't very useful and broke whole page on IE

**Pull request merged**